test/e2e/fast-prototyping/reactors/mempool/gossip/mempoolTx.go

```diff
@@ -0,0 +1,34 @@
+package gossip
+
+import (
+	"sync"
+	"sync/atomic"
+
+	"github.com/cometbft/cometbft/types"
+)
+
+// mempoolTx is an entry in the mempool
+type mempoolTx struct {
```

Comment:
This type is based on the `mempool.Entry` time, apparently. The 3 first fields are identical. Something worth to mention.

Reply:
This type is no more used with the changes introduced in #1043. The file was removed.